stream_alert/rule_processor/parsers.py

```diff
@@ -104,19 +104,28 @@ def _key_check(self, json_records):
         """
         schema = self._get_schema()
         schema_keys = set(schema.keys())
+        valid_records = []
+        schema_match = False
```

Comment:
As discussed offline, this solution does not account for string/integer values in the schema.